lisa/sut_orchestrator/azure/features.py

```diff
@@ -1367,8 +1367,12 @@ def _select_vm_size(
         # Choose random size from the list to resize to
         index = randint(0, len(avail_eligible_intersect) - 1)
         resize_vm_size = avail_eligible_intersect[index]
+        origin_vm_size = node_runbook.vm_size
+        node_runbook.vm_size = resize_vm_size.vm_size
+        node_runbook.location = resize_vm_size.location
+        resize_vm_size.capability.set_extended_runbook(node_runbook)
         self._log.info(f"New vm size: {resize_vm_size.vm_size}")
-        return resize_vm_size
+        return resize_vm_size, origin_vm_size
```

Comment:
origin, then new size.

Reply:
this is used in https://github.com/microsoft/lisa/pull/2438/files#diff-05e1ad7a12a81a2eff6db4a7842324e2c04cbde8cf304116e8b072cc16bdd13bR1258

and here https://github.com/microsoft/lisa/pull/2438/files#diff-05e1ad7a12a81a2eff6db4a7842324e2c04cbde8cf304116e8b072cc16bdd13bR1277 used in test case

Reply:
I mean, swap their positions. The order old/new is more natural than new/old.

```suggestion
        return origin_vm_size, resize_vm_size
```